app/js/components/tabs/tabcomponents/observationComponent.jsx

```diff
@@ -21,7 +21,7 @@ export default class ObsFilter extends React.Component {
     this.state = {
       timeModifier: 'ANY',
       question: props.concept.uuid,
-      operator: '',
+      operator1: 'LESS_THAN',
```

Comment:
Why not just assign 'LESS THAN' to operator?

Reply:
As stated by Mike on this [thread](https://talk.openmrs.org/t/how-do-i-format-query-request-when-querying-for-observations-in-cohort-builders-concept-observation/13883/2),  operator is an invalid parameter name. operator1 and operator2 should be used